index.bs

```diff
@@ -1171,19 +1173,19 @@ handler.
 
   An [=/upgrade transaction=] is exclusive. The steps to [=open a
   database=] ensure that only one [=/connection=] to the database is
-  open when an [=/upgrade transaction=] is running.
+  open when an [=/upgrade transaction=] is [=transaction/live=].
   The {{IDBOpenDBRequest/upgradeneeded!!event}} event isn't fired, and thus the
   [=/upgrade transaction=] isn't started, until all other
   [=/connections=] to the same [=/database=] are closed. This ensures
   that all previous transactions are [=transaction/finished=]. As long
-  as an [=/upgrade transaction=] is running, attempts to open more
+  as an [=/upgrade transaction=] is [=transaction/live=], attempts to open more
   [=/connections=] to the same [=/database=] are delayed, and any
   attempts to use the same [=/connection=] to start additional
   transactions by calling {{IDBDatabase/transaction()}} will throw an
   exception. Thus [=/upgrade transactions=] not only ensure that no
```

Comment:
Whee, this text basically goes back to 2012. At least it's non-normative now. :) I rephrased a bit, I think the new form is clearer? As it is non-normative, I think the whole sentence could easily be dropped. But presumably it was added to make something explicit to a reader at some point. WDYT?

Reply:
I agree that dropping it might ultimately be less confusing since it's essentially restating information from elsewhere in the doc, but less clearly (imo). The new form is better too.